scripts/travis.sh

```diff
@@ -12,5 +13,8 @@ if [ "${TRAVIS_EVENT_TYPE}" = "cron" ] ; then
     fi
     yarn test:e2e
 else
+    if [[ "$TEST_SUITE" == *"e2e:windows"* ]] ; then
```

Comment:
Only `test:e2e:windows` test suite uses saucelabs.
The regular e2e tests are run on chrome headless